ci-operator/step-registry/windows/ci/fastforward/windows-ci-fastforward-commands.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+wmco_dir=$(mktemp -d -t wmco-XXXXX)
+cd "$wmco_dir" || exit 1
+export HOME="$wmco_dir"
+
+echo "INFO Checking settings"
+echo "    REPO_OWNER         = $REPO_OWNER"
+echo "    REPO_NAME          = $REPO_NAME"
+echo "    JOB_TYPE           = $JOB_TYPE"
+echo "    GITHUB_USER        = $GITHUB_USER"
+echo "    GITHUB_TOKEN_FILE  = $GITHUB_TOKEN_FILE"
+echo "    SOURCE_BRANCH      = $SOURCE_BRANCH"
+echo "    DESTINATION_BRANCH = $DESTINATION_BRANCH"
+
+if [[ "$JOB_TYPE" != "postsubmit" ]]; then
+    echo "ERROR This workflow may only be run as a postsubmit job"
+    exit 1
+fi
+
+if [[ -z "$GITHUB_USER" ]]; then
+    echo "ERROR GITHUB_USER may not be empty"
+    exit 1
+fi
+
+if [[ ! -r "${GITHUB_TOKEN_FILE}" ]]; then
+    echo "ERROR GITHUB_TOKEN_FILE missing or not readable"
+    exit 1
+fi
+
+if [[ -z "$SOURCE_BRANCH" ]]; then
+    echo "ERROR SOURCE_BRANCH may not be empty"
+    exit 1
+fi
+
+if [[ -z "$DESTINATION_BRANCH" ]]; then
+    echo "ERROR DESTINATION_BRANCH may not be empty"
+    exit 1
+fi
+
+echo "INFO Cloning $DESTINATION_BRANCH"
+token=$(cat "$GITHUB_TOKEN_FILE")
+repo="github.com/$REPO_OWNER/$REPO_NAME"
+repo_url="https://${GITHUB_USER}:${token}@${repo}.git"
+
+if ! git clone -b "$DESTINATION_BRANCH" "$repo_url" ; then
+    echo "INFO $DESTINATION_BRANCH does not exist. Will create it"
```

Comment:
Does this mean we no longer have to manually created `community-4.*` branches?

Reply:
Yes, is that ok? Are there steps missing that you usually follow when creating a community branch?

Reply:
That should be ok. I guess we can find out by deleting the community-4.13 branch after this PR merges 😄